cassandra_backup_service.py

```diff
@@ -861,11 +861,14 @@ def __enumerate_cluster(self):
                 datacenter = line.split('Datacenter: ')[1]
                 self.cluster_data[datacenter] = {}
             else:
-                # TODO: When a node is down the load quantity is "?" and load units is missing.
                 version = get_version()
                 if '?' in line and version[0] == '2':
                     line = line.replace('?', '? ?')
 
+                # When a node is down the load quantity is "?" and load units is missing.
+                if '?' in line and len(line.split()) == 7:
```

Comment:
The change is still failing because major version is till 2 and line already have 8 tokens in it. so code on line # 865 increase the token to 9 and fails again.
I think along with the change add this to following on line # 865
if '?' in line and version[0] == '2': and len(line.split()) == 7